taskswamp

```diff
@@ -85,9 +85,14 @@ def main():
 
     assert config['taskswamp'] == -1, "Missing config format number"
 
+    refresh_every = "0"
+    if 'refresh_every' in config:
+        refresh_every = config['refresh_every']
+    assert int(refresh_every) >= 0
```

Comment:
Please document the new config key.  (Note that sleep(1) supports fractional arguments on some system, but POSIX only requires it to support integers.)

As written this is going to result in a ValueError exception if the config value is set and is not `int()`-able.  Should we try for a nicer error message?

If the value is set to a negative number the user will get an assertion error.  Could you add a message in that case, like "Invalid configuration" in the previous hunk?

Reply:
I documented the new key in the readme in 9c3b9375e54

Reply:
Gentler error handling is in 73003bb